BuildAddOn.py

```diff
@@ -290,9 +290,11 @@ def PackageAddOns (args, addOnName, platformName, acVersionList, languageList, b
             CopyResultToPackage (packageRootFolder, buildFolder, version, addOnName, platformName, 'RelWithDebInfo')
         
         buildType = 'Release' if args.release else 'Daily'
+        buildNum = configData["devKitLinks"][platformName][version].split ('.')[-2]

```

Comment:
This introduces a limitation on the name of the DevKit, it won't work if I use a custom url.

Reply:
Fixed